akka-actor-testkit-typed/src/test/java/akka/actor/testkit/typed/javadsl/BehaviorTestKitTest.java

```diff
@@ -111,68 +110,96 @@ public SpawnChildrenAnonymousWithProps(int numberOfChildren, Props props) {
     }
   }
 
-  public interface Action {
-  }
+  public interface Action {}
 
   private static Behavior<Action> childInitial = Behaviors.ignore();
 
   private static Props props = Props.empty().withDispatcherFromConfig("cat");
 
-  private static Behavior<Command> behavior = Behaviors.receive(Command.class)
-    .onMessage(SpawnChildren.class, (ctx, msg) -> {
-      IntStream.range(0, msg.numberOfChildren).forEach(i -> {
-        ctx.spawn(childInitial, "child" + i);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(SpawnChildrenAnonymous.class, (ctx, msg) -> {
-      IntStream.range(0, msg.numberOfChildren).forEach(i -> {
-        ctx.spawnAnonymous(childInitial);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(SpawnChildrenWithProps.class, (ctx, msg) -> {
-      IntStream.range(0, msg.numberOfChildren).forEach(i -> {
-        ctx.spawn(childInitial, "child" + i, msg.props);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(SpawnChildrenAnonymousWithProps.class, (ctx, msg) -> {
-      IntStream.range(0, msg.numberOfChildren).forEach(i -> {
-        ctx.spawnAnonymous(childInitial, msg.props);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(CreateMessageAdapter.class, (ctx, msg) -> {
-      ctx.messageAdapter(msg.clazz, msg.f);
-      return Behaviors.same();
-    })
-    .onMessage(SpawnWatchAndUnWatch.class, (ctx, msg) -> {
-      ActorRef<Action> c = ctx.spawn(childInitial, msg.name);
-      ctx.watch(c);
-      ctx.unwatch(c);
-      return Behaviors.same();
-    })
-    .onMessage(SpawnAndWatchWith.class, (ctx, msg) -> {
-      ActorRef<Action> c = ctx.spawn(childInitial, msg.name);
-      ctx.watchWith(c, msg);
-      return Behaviors.same();
-    })
-    .onMessage(SpawnSession.class, (ctx, msg) -> {
-      ActorRef<String> session = ctx.spawnAnonymous(Behaviors.receiveMessage( m -> {
-        msg.sessionHandler.tell(m);
-        return Behaviors.same();
-      }));
-      msg.replyTo.tell(session);
-      return Behaviors.same();
-    })
-    .onMessage(KillSession.class, (ctx, msg) -> {
-      ctx.stop(msg.session);
-      msg.replyTo.tell(Done.getInstance());
-      return Behaviors.same();
-    })
-    .build();
-
+  private static Behavior<Command> behavior =
+      Behaviors.receive(Command.class)
+          .onMessage(
```

Comment:
I think it would be better to use 2 spaces, also in Java

Reply:
Yes, the default one is 4 spaces.

Reply:
Kind of weird: the default indent for blocks is 2 spaces which is good, but for line breaks apparently it's 4? Can this be customized?

Reply:
```
Note: There is no configurability as to the formatter's algorithm for formatting. This is a deliberate design decision to unify our code formatting on a single format.
```
:(